packages/devkit/src/utils/package-json.ts

```diff
@@ -0,0 +1,161 @@
+import { readFileSync } from 'fs';
+import { lt, valid } from 'semver';
+
+import { readJson, updateJson } from './json';
+import { Tree } from '@nrwl/tao/src/shared/tree';
+import { installPackagesTask } from '../tasks/install-packages-task';
+
+/**
+ * Add Dependencies and Dev Dependencies to package.json and queue an npm install if necessary
```

Comment:
I'd not use the word queue cause we aren't queueing anything. Maybe say and returns a task that runs npm installs

Reply:
I'd also add a small example